### ISSUE TITLE
Support for controlling input_boolean is only partly implemented

### ISSUE THREAD

**Issue by skannea** [xaviml/controllerx#146]
# Bug report
  
## Description
It is not clearly said in the documentation that you can control an 'input_boolean' as if it were a 'switch'. However, the domains  _"switch", "input_boolean", "binary_sensor"_ are all allowed according to the code. 

The problem is that SwitchController class methods _on_, _off_ and _toggle_ are hard coded to the services "switch/turn_on" and so on.
For example:
```
    @action
    async def on(self) -> None:
        await self.call_service("switch/turn_on", entity_id=self.switch)
```

This works only for the switch domain. 
I made this modification to make them work, but there might be some better way:
```    
    @action
    async def on(self) -> None:
        await self.call_service( self.switch.split('.')[0] + "/turn_on", entity_id=self.switch)
```

## AppDaemon app configuration

```
conbee_1:
  module: controllerx
  class: E1743SwitchController
  controller: tradfri_on_off_switch
  integration: deconz  
  switch: input_boolean.epsilon
```

**Comment by xaviml** [comment]
Hi @skannea,

Thank you for spotting this.

Regarding that SwitchControkker can be used for `input_boolean` and `binary_sensor` (not sure why I support this one since it cannot be modified) is in [here](https://xaviml.github.io/controllerx/start/type-configuration#switch-controller).

Regarding the call service, it could be changed to `homeassistant.toggle` service which is a generic service that toggles all these entities. I will change this for next release.

Cheers,
Xavi M.

**Comment by xaviml** [comment]
This has been fixed in ControllerX v4.0.0. Now you can use switch controllers for anything that turns on/off and toggle (alert, automation, cover, input_boolean, light, media_player, script, switch).
